plugins/modules/remote_management/redfish/redfish_config.py

```diff
@@ -253,7 +269,8 @@
 CATEGORY_COMMANDS_ALL = {
     "Systems": ["SetBiosDefaultSettings", "SetBiosAttributes", "SetBootOrder",
                 "SetDefaultBootOrder"],
-    "Manager": ["SetNetworkProtocols", "SetManagerNic", "SetHostInterface"]
+    "Manager": ["SetNetworkProtocols", "SetManagerNic", "SetHostInterface"],
+    "SessionService": ["SetSessionService"]
```

Comment:
The category should be named "Sessions" to be consistent with the category naming in redfish_command.py and redfish_info.py. "SetSessionService" is good for the specific command name.